macro/Version.py

```diff
@@ -36,16 +55,40 @@ def execute(macro, args):
                     'New in %s</span>' % version)
 
     def distro_html(distro, distros):
-        active = [distro.encode("iso-8859-1")]
-        inactive = [x.encode("iso-8859-1") for x in distros if not x == distro]
+        html = ""
+        distro_found = False
+        preceding_distros = [distro]
+        proceeding_distros = [distro]
+        for d in sorted(distros):
+            if d == distro:
+                distro_found = True
+                continue
+            if distro_found:
+                proceeding_distros.append(d)
+            if not distro_found:
+                preceding_distros.append(d)
```

Comment:
Instead of using a loop the found flag can be determined by:

    distro_found = distro in distros

And the proceedings / precedings can be determined by a range in the sorted list.